examples/nextjs/pages/posts/[slug].js

```diff
@@ -0,0 +1,50 @@
+import {
+  useContentfulInspectorMode,
+  useContentfulLiveUpdates,
+} from "@contentful/live-preview/react";
+import { useRouter } from "next/router";
+import Head from "next/head";
+import ErrorPage from "next/error";
+import { getAllPostsWithSlug, getPost } from "../../lib/api";
+
+export default function Post({ post, preview }) {
+  const router = useRouter();
+  const updatedPost = useContentfulLiveUpdates(post);
+  const inspectorProps = useContentfulInspectorMode({ entryId: post.sys.id });
+
+  if (!router.isFallback && !updatedPost) {
+    return <ErrorPage statusCode={404} />;
+  }
+
+  return (
+    <>
+      {updatedPost.title && (
```

Comment:
we could also render the `<h1>` but fallback to an empty string (if it is required in the contentType)